contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,78 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+import logging
+from google import search
+
+def get_google_result(original_content):
+    search_keywords = original_content.split(' ', 1)[1]
+    if search_keywords == 'help':
+        help_message = "To use this bot start message with @google followed \
+                        by what you want to search for. If found, Zulip will \
+                        return the first search result on Google. An example \
+                        message that could be sent is: '@google zulip' or '@google \
+                        how to create a chatbot'."
+        return help_message
+    else:
+        try:
+            urls = search(search_keywords, stop=20)
+        except:
```

Comment:
here you need to check what's exception is being raised by search function and then catch this particular kind. Then get its message and put inside the return parameter.